### ISSUE TITLE
Source installation (OS X) is possibly missing instructions related to prokaryote

### ISSUE THREAD

**Issue by shntnu** [CellProfiler/CellProfiler#1775]
I followed
https://github.com/CellProfiler/CellProfiler/wiki/*Source-installation-(OS-X)

I checked out 60f697e8, everything installs ok and CP starts fine, but when I load an image and try to view it, I get 

```
Failed to get class loci/common/RandomAccessInputStream
Exception in thread "Thread-1" java.lang.NoClassDefFoundError: loci/common/RandomAccessInputStream
Caused by: java.lang.ClassNotFoundException: loci.common.RandomAccessInputStream
	at java.net.URLClassLoader$1.run(URLClassLoader.java:372)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:361)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:360)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:308)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
```

And this is the log upon starting CP (note the "Did not find prokaryote")

```
Did not find prokaryote in ['/Users/shsingh/Library/Python/2.7/lib/python/site-packages/javabridge/jars/rhino-1.7R4.jar', '/Users/shsingh/Library/Python/2.7/lib/python/site-packages/javabridge/jars/runnablequeue.jar', '/Users/shsingh/Library/Python/2.7/lib/python/site-packages/javabridge/jars/cpython.jar', '/Users/shsingh/work/software/CP/shntnu/CellProfiler/plugins']
Exception in thread "Thread-1" java.lang.ClassNotFoundException: loci.common.Location
	at java.net.URLClassLoader$1.run(URLClassLoader.java:372)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:361)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:360)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:308)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:344)
Bioformats version does not support directory cacheing
vigra import: failed to import the vigra library. Please follow the instructions on
"http://hci.iwr.uni-heidelberg.de/vigra/" to install vigra
Traceback (most recent call last):
  File "cellprofiler/modules/classifypixels.py", line 50, in <module>
    import vigra
ImportError: No module named vigra
vigra import: failed to import the vigra library. Please follow the instructions on
"http://hci.iwr.uni-heidelberg.de/vigra/" to install vigra
Traceback (most recent call last):
  File "cellprofiler/modules/ilastik_pixel_classification.py", line 47, in <module>
    import vigra
ImportError: No module named vigra
Version: 2016-02-04T18:21:55 60f697e / 20160204182155
```

**Comment by LeeKamentsky** [comment]
Have you run `pip install --editable $HOME/CellProfilar --user` since cloning? I am guessing that somehow prokaryote hasn't been installed. Can you post the results of `pip freeze` and `python -c "import prokaryote;print prokaryote.__version__"? Here is a work-around:

    pip uninstall prokaryote
    pip install prokaryote

**Comment by shntnu** [comment]
@LeeKamentsky This seems to be fixed in dc5df4536fddf7cadc2c86041cf04887d87d1404 

```
$ pip freeze
-e git+https://github.com/CellH5/cellh5.git@3c1d6ea473ddb9065538b0da1462f117ce571f3a#egg=cellh5
-e git+https://github.com/shntnu/CellProfiler@dc5df4536fddf7cadc2c86041cf04887d87d1404#egg=CellProfiler
centrosome==1.0.2
cycler==0.9.0
Cython==0.23.4
decorator==4.0.4
fix-osx-virtualenv==0.1
h5py==2.5.0
javabridge==1.0.11
libtiff==0.4.0
lxml==3.5.0
matplotlib==1.5.0
MySQL-python==1.2.5
networkx==1.10
numpy==1.10.1
pandas==0.17.0
Pillow==3.0.0
prokaryote==1.0.8
py==1.4.31
pyparsing==2.0.6
pytest==2.8.7
python-bioformats==1.0.5
python-dateutil==2.4.2
pytz==2015.7
pyzmq==14.7.0
scikit-image==0.11.3
scikit-learn==0.16.1
scipy==0.16.1
six==1.10.0
wheel==0.24.0
wxPython==3.0.2.0
$python -c "import prokaryote;print prokaryote.__version__"
1.0.8
```

**Comment by zindy** [comment]
Is it OK to reopen this? I have the exact same issue in Linux (with a local install of Anaconda):

```
[epzindy@mymachine CellProfiler]$ python CellProfiler.py
Did not find prokaryote in ['/mnt/home/epzindy/anaconda/lib/python2.7/site-packages/javabridge/jars/rhino-1.7R4.jar', '/mnt/home/epzindy/anaconda/lib/python2.7/site-packages/javabridge/jars/runnablequeue.jar', '/mnt/home/epzindy/anaconda/lib/python2.7/site-packages/javabridge/jars/cpython.jar', '/mnt/home/epzindy/projects/CellProfiler/plugins']
Exception in thread "Thread-0" java.lang.ClassNotFoundException: loci.common.Location
	at java.net.URLClassLoader$1.run(URLClassLoader.java:366)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:355)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:354)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:425)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:308)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:358)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:278)
Bioformats version does not support directory cacheing
```
And as requested to @shntnu 

```
[epzindy@mymachine CellProfiler]$ pip freeze
abstract-rendering==0.5.1
alabaster==0.7.7
anaconda-client==1.2.2
argcomplete==1.0.0
args==0.1.0
astropy==1.1.1
Babel==2.2.0
backports-abc==0.4
backports.ssl-match-hostname==3.4.0.2
bcolz==0.0.0
beautifulsoup4==4.4.1
binstar==0.10.1
bitarray==0.8.1
blaze==0.9.0
bokeh==0.11.0
boto==2.39.0
Bottleneck==1.0.0
cdecimal==2.3
cellh5==1.3.0
-e git+https://github.com/CellProfiler/CellProfiler.git@4ae41bce0acf62333c317a71b1f41f64b66d3bad#egg=CellProfiler
centrosome==1.0.4
certifi==14.5.14
cffi==1.2.1
clint==0.5.1
clyent==1.2.0
colorama==0.3.6
conda==4.0.5
conda-build==1.11.0
conda-env==2.4.5
configobj==5.0.6
Contrast-stretching-for-numpy-arrays==1.0
cryptography==0.9.1
cycler==0.9.0
Cython==0.23.4
cytoolz==0.7.5
datashape==0.5.0
decorator==4.0.6
docutils==0.12
dynd===f641248
enum34==1.1.2
et-xmlfile==1.0.1
fastcache==1.0.2
Flask==0.10.1
funcsigs==0.4
future==0.15.2
futures==3.0.3
gevent==1.0.2
gevent-websocket==0.9.5
greenlet==0.4.9
grin==1.2.1
h5py==2.5.0
hmmlearn==0.2.0
idna==2.0
inflect==0.2.5
ipaddress==1.0.14
ipykernel==4.2.2
ipython==4.0.3
ipython-genutils==0.1.0
ipywidgets==4.1.1
itsdangerous==0.24
javabridge==1.0.11
jdcal==1.2
jedi==0.9.0
Jinja2==2.8
jsonschema==2.4.0
jupyter==1.0.0
jupyter-client==4.1.1
jupyter-console==4.1.0
jupyter-core==4.0.6
libtiff==0.4.0
llvmlite==0.8.0
lxml==3.5.0
MarkupSafe==0.23
matplotlib==1.5.1
mistune==0.7.1
mock==1.0.1
multipledispatch==0.4.8
MySQL-python==1.2.5
nbconvert==4.1.0
nbformat==4.0.1
networkx==1.11
nltk==3.1
nose==1.3.7
notebook==4.1.0
numba==0.23.1
numexpr==2.4.6
numpy==1.10.4
odo==0.4.0
openpyxl==2.3.2
pandas==0.17.1
path.py==0.0.0
patsy==0.4.0
pep8==1.7.0
pexpect==3.3
pickleshare==0.5
Pillow==3.1.0
ply==3.8
prokaryote==1.0.10
psutil==3.4.2
ptyprocess==0.5
py==1.4.31
pyasn1==0.1.9
pycairo==1.10.0
pycosat==0.6.1
pycparser==2.14
pycrypto==2.6.1
pycurl==7.19.5.3
pyfaidx==0.4.2
pyflakes==1.0.0
Pygments==2.1
pyOpenSSL==0.15.1
pyparsing==2.0.3
pytest==2.8.5
python-bioformats===1.0.5-29-g2f10961
python-dateutil==2.4.2
pytz==2015.7
PyVCF==0.6.7
PyYAML==3.11
pyzmq==15.2.0
qtconsole==4.1.1
ratiolib==1.0
redis==2.10.3
requests==2.9.1
rope==0.9.4
runipy==0.1.3
scikit-image==0.11.3
scikit-learn==0.17
scipy==0.17.0
simplegeneric==0.8.1
singledispatch==3.4.0.3
six==1.10.0
snowballstemmer==1.2.1
sockjs-tornado==1.0.1
Sphinx==1.3.5
sphinx-rtd-theme==0.1.9
spyder==2.3.8
SQLAlchemy==1.0.11
statsmodels==0.6.1
sympy==0.7.6.1
tables==3.2.2
terminado==0.5
Theano==0.7.0
tifffile==2014.8.24
toolz==0.7.4
tornado==4.3
traitlets==4.1.0
ujson==1.33
unicodecsv==0.14.1
Werkzeug==0.11.3
wheel==0.26.0
wxPython==3.0.0.0
wxPython-common==3.0.0.0
xlrd==0.9.4
XlsxWriter==0.8.4
xlwt==1.0.0

[epzindy@mymachine CellProfiler]$ python -c "import prokaryote;print prokaryote.__version__"
1.0.10
```

Do let me know if there is anything I can test.

Kind regards,
Egor
